### PR TITLE
[dom-gpu] googletest

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Comment by jgphpc** [comment]
retest this please

**Comment by jgphpc** [comment]
retest this please

File: easybuild/easyconfigs/g/googletest/googletest-1.8.1-CrayGNU-18.08.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['release-%(version)s.tar.gz']
+checksums = ['2e6fbeb6a91310a16efe181886c59596']
+
+builddependencies = [
+    ('CMake', '3.12.0', '', True),
+]
+
+separate_build_dir = True
+
+configopts  = '-DCMAKE_CXX_COMPILER=`which g++` -DCMAKE_C_COMPILER=`which gcc` '
```
Comment (victorusu) [inline easybuild/easyconfigs/g/googletest/googletest-1.8.1-CrayGNU-18.08.eb]: What about using the cray compiler wrappers?

Comment (jgphpc) [reply easybuild/easyconfigs/g/googletest/googletest-1.8.1-CrayGNU-18.08.eb]: in this special case, i also want to be able to run on the login nodes too but if i compile with the cray wrappers, i won't be able to.

Comment (victorusu) [reply easybuild/easyconfigs/g/googletest/googletest-1.8.1-CrayGNU-18.08.eb]: Ok, then! 👍

**Pull request merged**